src/Chainweb/Version/Registry.hs

```diff
@@ -85,9 +86,7 @@ validateVersion v = do
                     , hasAllChains (_genesisTime $ _versionGenesis v)
                     ])]
             ]
-    if null errors
-    then return ()
-    else
+    unless (null errors) $
```

Comment:
Same suggestion as above;
```suggestion
    when (not (null errors)) $
```

Reply:
Oof I like this one too much as `unless`.